### ISSUE TITLE
Recreate table log_link_visit_action

### ISSUE THREAD

**Issue by Harfeur** [piwik/piwik#20172]
## Expected Behavior

I should see the dashboard

## Current Behavior

SQLSTATE[42S02]: Base table or view not found: 1146 Table 'matomo_db.matomo_log_link_visit_action' doesn't exist

## Possible Solution

Recreate the table, but I can't find the script and the one in core/Db/Schema/Mysql.php is incomplete

## Steps to Reproduce (for Bugs)

I don't know

## Context

I tried to add a new website to my instance

## Your Environment

* Matomo Version: 4.13.0
* PHP Version: 8.0.26
* Server Operating System: Debian GNU/Linux 11
* Additionally installed plugins: Nope

**Comment by bx80** [comment]
Hi @Harfeur, thanks for reaching out.

It would be good to first check the database config settings you have in `config.ini.php` and if possible manually connect to confirm that the database exists, is accessible and what tables are present.

If one or more tables is missing from your database then I'd recommend restoring from a database backup as the remaining tables may now reference data that is no longer present and this will likely cause unpredictable errors.

**Comment by Harfeur** [comment]
Hi, thanks for your quick reply.

> manually connect to confirm that the database exists, is accessible and what tables are present.

I connected to the database but this specific wasn't in the database.

> restoring from a database backup

Unfortunately I didn't have a database backup, so I created a new matomo instance and I followed the install guide that created the missing table. Then I got the table data and I recreated the table in the original database.

Now the problem is solved, thanks !
